posthog/settings/web.py

```diff
@@ -96,6 +96,7 @@
     "django_structlog.middlewares.RequestMiddleware",
     "django_structlog.middlewares.CeleryMiddleware",
     "django.middleware.security.SecurityMiddleware",
+    "whitenoise.middleware.WhiteNoiseMiddleware",
```

Comment:
I would definitely put this behind ShortCircuitMiddleware, as otherwise you're loading this for every capture/decide endpoint hit.